assets/out

```diff
@@ -60,11 +61,16 @@ if [ -n "$tag" ]; then
   tag_name="$(cat $tag)"
 fi
 
+annotate=""
+if [ -n "$message" ]; then
+  annotate=" -a -m '$(cat $message)'"
```

Comment:
I'm afraid of interpolating messages like this; seems like it would break if their message had a `'` in it.

Looks like there's a `-F` flag for reading the message from a file, so this could become:

```sh
  annotate=" -a -F '$message'"
```